tests/bundle-test.js

```diff
@@ -80,13 +79,13 @@ async function runAll() {
   console.log('bob setup done')
 
   // Light Did Account creation workflow
-  const authPublicKey = Crypto.coToUInt8(
+  const authPublicKey = window.kilt.Utils.Crypto.coToUInt8(
```

Comment:
The current tsdoc version requires me to include the DOM lib, because its using highlight.js which uses stuff like `HTMLElement`. When including the DOM lib I got an error when compiling this file because `Crypto` and one other thing were already defined and typescript refused to re-declare them globally.

The simplest solution is to just use them in a fully qualified way.

Reply:
I fixed it